agency/README.md

```diff
@@ -259,6 +260,56 @@ A standard point of vehicle telemetry. References to latitude and longitude impl
 
 [Top][toc]
 
+## Trips
+
+The `/trips` endpoint is a subset of the similar endpoint in Provider, except it's push instead of pull.  It receives trip information that is above and beyond what is encoded in the register, event and telemetry streams.
```

Comment:
Why is it only a subset? Might be good to include in the spec.

Reply:
I think that's what the second sentence is saying, but maybe there needs to be more language to elaborate upon it. Provider doesn't have a notion of telemetry (closest thing is `route` in a the Provider `/trips` endpoint), and it also doesn't have an always-present vehicle registry that can be polled (the Provider `/vehicles` endpoint only returns vehicles in the PROW), so there are some additional pieces of info that need to be in the Provider `/trips` endpoint that are unnecessary to send to Agency because Agency already has them.